src/video/windows/SDL_windowsrawinput.c

```diff
@@ -183,7 +184,7 @@ static int WIN_UpdateRawInputEnabled(SDL_VideoDevice *_this)
 {
     SDL_VideoData *data = _this->driverdata;
     Uint32 flags = 0;
-    if (data->raw_mouse_enabled) {
+    if (data->raw_mouse_enabled || SDL_EventEnabled(SDL_EVENT_MOUSE_RAW_MOTION)) {
```

Comment:
You probably need a way to re-evaluate this when the event enable state changes.

Reply:
Enabling the event shouldn't turn on relative mouse mode, as that has a bunch of other side effects you want to avoid here, right?

Reply:
I would probably add a private video function that updates raw mouse motion generation state. It's fine for code in one part of SDL to call non-platform specific functions in another part of SDL. events -> video and vice versa, for example.

Reply:
I would still have a separate enable path that doesn't have any of the side effects of relative mouse mode.

Reply:
Exactly, but I would probably make it specific to raw mouse motion rather than "PlatformInputSubscription"